src/main/resources/static/html/header.html

```
@@ -1,14 +1,144 @@
+<!-- 1) 기존 header.css 로딩 -->
+<link rel="stylesheet" href="/css/header.css" />
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)